### ISSUE TITLE
validate and test set data

### ISSUE THREAD

**Issue by zllxot** [ucla-mobility/V2V4Real#3]
Thank you for your great contribution! I have downloaded the LiDAR + Labels (OPV2V format) (train1~train8) files, and I guess they should be unzipped to `v2v4real/train` directory, so how to obtain the `validate` and `test` set data?

**Comment by DerrickXuNu** [comment]
you are correct about train. Are you able to visualize the train dataset?
In terms of validate and test, they will be uploaded shortly

**Comment by zllxot** [comment]
> you are correct about train. Are you able to visualize the train dataset? In terms of validate and test, they will be uploaded shortly

Yes, I have successfully visualized the training set and I am looking forward to the release of the test set and validate set.

**Comment by DerrickXuNu** [comment]
The test set is released

**Comment by zllxot** [comment]
thanks. By the way, when will the validate set be released?
